prow/config.yaml

```diff
@@ -4935,6 +4965,29 @@ postsubmits:
       - name: docker-graph
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
+
+  kubernetes/kube-deploy:
```

Comment:
I would add them as periodic jobs rather than postsubmits?

Reply:
Ok. What is the purpose of post-submits? Just stuff you don't want to run on every commit in every PR?

Reply:
It's the opposite right? Postsubmits are the stuff you do want to run on every commit in every PR :-)

there are jobs like config updater, need to run right after a push event...

Reply:
What does this job do exactly? post-submits are best used for things like uploading a new config after the config change is merged, IE some post-merge publish step. 

Continuous tests are probably best as periodics and/or presubmits.

Reply:
Got it. It's just CI builds... periodic it is.